server/test/database.spec.js

```diff
@@ -0,0 +1,56 @@
+const assert = require("chai").assert;
+const expect = require("chai").expect;
+const path = require("path");
+const mongoose = require("mongoose");
+const { connectDB, disconnectDB } = require("../utils/database");
```

Comment:
So at the moment it seems like you're not connecting or disconnecting from the database when performing these tests:  I might be missing something, but will these tests run successfully as written right now?

Reply:
Yes, I'm just testing the user model itself, not the connection to the database. My understanding from the stand-up this morning was that testing the connection itself would not make sense because it's out of our control.
The tests all pass as they are right now.
![passing-tests](https://user-images.githubusercontent.com/31010488/107585082-93f87980-6bcb-11eb-958e-13b75bf2515a.png)

Reply:
Oh, you won't want tests to check the connection and disconnection functions, but you will want to test adding/removing data from the database!

Reply:
Got it! I didn't create the functions for doing CRUD operations yet, because it wasn't on my assigned issue (#41) (but I'd love to do that if you want to assign me that task!)